rpcd/playbooks/rpc-support.yml

```diff
@@ -18,3 +18,12 @@
   user: root
   roles:
     - { role: "rpc_support", tags: [ "rpc-support" ] }
+
+- name: Drop pccommon script
```

Comment:
presumably at some point this role will do more than just install pccommon from it's main.yml? I'd suggest a more generic name for this play (e.g. 'install support scripts'), and the task inside the role should be named for what it's doing (i.e. 'install pccommon script' or whatever).